tests/_support/Page/plan/Create.php

```diff
@@ -11,10 +11,12 @@
 namespace hipanel\modules\finance\tests\_support\Page\plan;
 
 use hipanel\helpers\Url;
+use hipanel\tests\_support\Helper\CurrencyListTrait;
```

Comment:
сорі додай цей трейт в цей пр

```suggestion
use hipanel\module\finance\tests\_support\Helper\CurrencyListTrait;
```